api/v4/source/users.yaml

```diff
@@ -271,6 +271,8 @@
 
             When left blank, sorting is done by username.
 
+            Note that when "last_activity_at" is specified, an additional "last_activity_at" field will be returned in the response packet.
```

Comment:
Interesting, so we don't return this metadata unless this particular sort field is specified?

Reply:
Correct.  It had me puzzled for a while, and I tripped over the answer almost by accident! 😎